src/ansys/pytwin/evaluate/tbrom.py

```diff
@@ -171,12 +165,47 @@ def _reduce_field_input(self, name: str, snapshot: Union[str, Path, np.ndarray])
                 self._infmcs[name][item] = mc[index]
                 index = index + 1
 
+    def update_output_field(self):
+        """
+        Compute the output field results with current mode coefficients.
+        """
+        mc = list(self._outmcs.values())
+        self._pointsdata[self.field_output_name] = mc[0] * self._pointsdata["mode" + str(1)]
+        for i in range(1, len(mc)):
+            self._pointsdata[self.field_output_name] = (
+                self._pointsdata[self.field_output_name] + mc[i] * self._pointsdata["mode" + str(i + 1)]
+            )
+        self._pointsdata.set_active_scalars(self.field_output_name)
```

Comment:
the PolyData object might contain several fields (of scalars, vectors,...). In our case, it contains all the modes fields as well as the output field characterized by the TBROM. This particular line is to set the default scalar field to the one related to the TBROM output field so that when we want to visualize the data, it automatically displays the field of interest (and not e.g. one of the modes fields). This can of course be changed with PyVista APIs by the user if needed (after retrieving the object with 'get_tbrom_output_field').